include/pdal/GDALUtils.hpp

```diff
@@ -46,7 +47,7 @@
 #include <vector>
 
 #include <cpl_port.h>
-#include <gdal.h>
+#include <gdal_priv.h>
```

Comment:
That's where the definitions are.  Seemed weird to me.  That's what the examples show.